src/App.js

```diff
@@ -1,13 +1,12 @@
 import './App.css';
 
 import Calculator from './components/Calculator';
+import Header from './components/Header';
 
 function App() {
   return (
     <div className="App">
-      <header>
-        <h1>Math Magicians</h1>
-      </header>
+      <Header />
       <Calculator />
     </div>
   );
```

Comment:
- As per [these](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Use%20the%20following,as%20the%20Calculator) requirements, you're to do the following:
   - [ ] Create a new Quote component.
   - [ ] User the given API to fetch data inside the Quote component using `useEffect`.
   - [ ] Create loading and error states inside of the component.
   - [ ] Import the Quote component inside of the App component.